docs/components/modeler/desktop-modeler/process-applications/process-application.md

```diff
@@ -0,0 +1,43 @@
+---
+id: process-application
+title: Process applications in Desktop Modeler
+description: In Desktop Modeler a process application is a folder that contains a .process-application file and a set of related files you can work on and deploy as a single bundle.
+---
+
+import OverlayImg from './img/overlay.png'
+
+In Desktop Modeler a [process application](../../../concepts/process-applications.md) is a folder that contains a `.process-application` file and a set of related files you can work on and deploy as a single bundle. These files define the automation logic but are often part of a larger project (e.g., a Java application).
```

Comment:
```suggestion
In Desktop Modeler a [process application](../../../concepts/process-applications.md) is a folder that contains a `.process-application` file and a set of Camunda 8 assets that you work on and deploy as a single bundle. Typically the process application also contains custom implementation logic, e.g. your job workers, implemented in a Java application.
```

Reply:
I find our use of so many different terminologies confusing. We have files, resources, assets. I'll leave it up to the technical writer to decide.